stylesheets/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules/
+/package-lock.json
```

Comment:
you didn't commit `package-lock.json`, is that intended?

Reply:
For a dev build, I prefer not to commit the lock file since it really doesn't matter. I'd rather detect any problems we might encounter in CI.

Reply:
OK 👍🏻